src/app/checkout/Checkout.tsx

```diff
@@ -502,7 +506,7 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
         }
 
         this.setState({ isRedirecting: true }, () => {
-            navigateToOrderConfirmation(window.location, orderId);
+            navigateToOrderConfirmation(window.location, orderId , isBuyNowCartEnabled);
```

Comment:
I think we should reorder the params since `isBuyNowCartEnabled` will always have a value. Where as orderId is optional.

Reply:
I have changed the parameter as an object, think this is the most future proof solution. Then it turns out Apple Pay requires all `navigateToOrderConfirmation` params to be optional ([see here](https://github.com/bigcommerce/checkout-js/blob/a7fa190d45b55500eeacb7d7c837f5b771578d85/src/app/customer/customWalletButton/ApplePayButton.tsx#L21)).